APMrover2/GCS_Mavlink.cpp

```diff
@@ -287,6 +287,16 @@ void Rover::send_wheel_encoder(mavlink_channel_t chan)
     }
 }
 
+void Rover::send_wheel_encoder_distance(mavlink_channel_t chan)
+{
+    // send wheel encoder data using wheel_distance message
+    if (g2.wheel_encoder.num_sensors() > 0) {
+    	double distance[sizeof(mavlink_wheel_distance_t::distance)/sizeof(double)] {};
```

Comment:
do we need a "double" here?  I very rarely see doubles in our code.

I think we normally use memcopy rather than std::copy.

Reply:
Mavlink message uses doubles, hence the doubles here and std::copy.